examples/goldilocks_models/find_goldilocks_models.cc

```diff
@@ -1567,6 +1594,27 @@ int findGoldilocksModels(int argc, char* argv[]) {
        << max_ground_incline << endl;
   cout << "turning rate ranges from " << min_turning_rate << " to "
        << max_turning_rate << endl;
+
+  /// How to restrict the number of samples is still under testing
```

Comment:
Is it still under testing? If not, we can update the comments here

Reply:
I am not sure whether we need to change the restricted number of samples while the task space keep increasing. So we might leave it until we finished increasing the task space.